README.md

```diff
@@ -1,3 +1,29 @@
+<p align="center">
```

Comment:
mmm, this does look nice rendered by github. I don't love dumping  html into our readme though. also the align attribute is [deprecated](https://www.geeksforgeeks.org/html-align-attribute/) in html5, so maybe not the best to use in new stuff?

I guess I don't generally love assuming people will only view this on github in the browser. I'd like `cat README.md` to work okay, which maybe it still does, but certainly not quite as well.

thoughts? 

If we gave up centering could the rest be done in "normal" markdown?